packages/middleware-sdk-ec2/src/index.ts

```diff
@@ -81,7 +76,7 @@ export const copySnapshotPresignedUrlMiddlewareOptions: InitializeHandlerOptions
   override: true,
 };
 
-export const getCopySnapshotPresignedUrlPlugin = (config: PreviouslyResolved): Pluggable<any, any> => ({
+export const getCopySnapshotPresignedUrlPlugin = (config: EC2ClientResolvedConfig): Pluggable<any, any> => ({
```

Comment:
https://github.com/aws/aws-sdk-js-v3/blob/main/clients/client-ec2/src/commands/CopySnapshotCommand.ts#L85 is the only caller, so this type was made more specific.